cypress/integration/smoke.spec.ts

```diff
@@ -26,7 +26,7 @@ function addCommonProvisioningADXDatasource(ADXProvisions: ADXProvision[]) {
   e2e.flows.addDataSource({
     type: 'Azure Data Explorer Datasource',
     form: () => {
-      e2eSelectors.configEditor.azureCloud.input().type('Azure');
+      e2eSelectors.configEditor.azureCloud.input().find('#azure-cloud-type').type('Azure');
```

Comment:
nit: I think `.find('input')` here is more stable rather than referencing the `htmlFor` field